schema-conformance/action.yaml

```diff
@@ -62,7 +62,18 @@ runs:
         export LOCAL_UID=`id -u`
         export LOCAL_GID=`id -g`
         mkdir $PWD/reports
-        alias levo="sudo docker run --rm --add-host=host.docker.internal:`ip route|awk '/docker0/ { print $9 }'` --mount type=bind,source=$HOME/.config/configstore,target=/home/levo/.config/configstore -v $PWD:/home/levo/work:rw -e LEVO_BASE_URL=${{ inputs.saas-url }} -e LOCAL_USER_ID=$LOCAL_UID -e LOCAL_GROUP_ID=$LOCAL_GID -e TERM=xterm-256color -t levoai/levo"
+        docker_engine_version=$(docker version -f '{{range .Server.Components}}{{if eq .Name "Engine"}}{{.Version}}{{end}}{{end}}')
+        IFS=. read major minor patch <<<"${docker_engine_version%%*-}"
+        if [[ $major -gt 20 ]] || [[ $major -eq 20 && $minor -gt 0 ]] || [[ $major -eq 20 && $minor -eq 0 && $patch -ge 4 ]]; then
+          docker_host_ip=host-gateway
+        else
+          docker_host_ip=`ip route|awk '/docker0/ { print $9 }'`
```

Comment:
@ricekot could this command fail in some cases?

Reply:
Yes, that's why I created this PR and why we have the fallback IP address.